### PR TITLE
Added responsive styles for search box. Fixes #250

### LINKED ISSUE #250: Make search responsive
The search isn't doing very well with small width

### PR DESCRIPTION
**Pull request by imaustink**
__Desktop:__
image

__Mobile:__
simulator screen shot may 24 2017 2 36 35 pm
simulator screen shot may 24 2017 3 26 25 pm

### CODE REVIEW

**Comment by justinbmeyer** [comment]
I noticed the `__testing__` and other markdown.  Is it possible to avoid that similar to how we are doing for the `title` attributes:

cursor_and_canjs_-_atm_guide
 

notice how it's not a problem there.

File: static/search.less [hunk 1]
```diff
@@ -151,3 +150,18 @@
...
+	&:after {
+		content:"";
+		position: absolute;
+		height: 100%;
+		width: 100%;
+		top: 0;
+		left: 0;
+		background-color: hsla(32, 96%, 49%, 0.61);
+	}
+}
```
Comment (chasenlehara) [inline static/search.less]: This should be taken out and only included in https://github.com/canjs/bit-docs-html-canjs/pull/261

**Comment by imaustink** [comment]
Updated the PR 😃

**Comment by imaustink** [comment]
@justinbmeyer good catch. I am sure it's possible, probably easy. For now, I opened another issue #264

**Comment by chasenlehara** [comment]
@imaustink In your screenshot, it looks like the results are over-escaped too. We have to fix those issues before we deploy search, so I’ve brought that issue you opened to the top of the backlog.

**Pull request merged**